.gitignore

```diff
@@ -7,6 +7,8 @@
 **/node_modules/**
 **/package.json
 **/package-lock.json
+**/alembic
+**/alembic.ini
```

Comment:
typically users will want to check in the migration scripts. i think adding these to `.gitignore` sets maybe a bad example.